ocrd_models/ocrd_models/ocrd_page_generateds.py

```diff
@@ -1229,7 +1229,7 @@ def __hash__(self):
     @property
     def id(self):
         if hasattr(self, 'pcGtsId'):
-            return self.pcGtsId
+            return self.pcGtsId or ''
```

Comment:
Is this an accidental change? It does not seem related to six?

Reply:
Ah, this is the generated file and behaviour seemed to have changed slightly. The question is if this is wanted or not

Reply:
Yes, this is intended, a change proposed by @bertsky to https://github.com/OCR-D/core/blob/master/ocrd_models/ocrd_page_user_methods/id.py which I forgot to regenerate the file for.